tests/bolt-testdata.hpp

```diff
@@ -261,9 +264,9 @@ std::vector<ValueTestParam> NodeTestCases() {
   std::vector<ValueTestParam> inputs;
 
   {
-    mg_string *labels[0] = {};
+    std::vector<mg_string *> labels(0);
```

Comment:
What kind of error did you get here?

Reply:
Oh, C++ doesn't like VLA, good to know.

Reply:
Yep, people even say VLAs are a huge mistake in the C standard...

Reply:
Actually, the problem here is a 0 sized array...